BGAnimations/ScreenSelectMusic overlay/PaneDisplay.lua

```diff
@@ -106,13 +106,27 @@ local GetScoresRequestProcessor = function(res, master)
 					end
 
 					if gsEntry["isSelf"] then
-						SetNameAndScore(
-							GetMachineTag(gsEntry),
-							string.format("%.2f%%", gsEntry["score"]/100),
-							playerName,
-							playerScore
-						)
-						personalRecordSet = true
+						-- Let's check if the GS high score is higher than the local high score
+						local player = i - 1
+						local localScore, localName
+
+						local SongOrCourse, StepsOrTrail = GetSongAndSteps(player)
```

Comment:
Can you break this out into a local function at the top of a file and re-use it at the bottom as well.

Reply:
Heh, funnily enough I did exactly that at first while prototyping, down to the function name :rofl: 
On it ^^